### ISSUE TITLE
Guile WM development

### ISSUE THREAD

**Issue by amirouche** [mwitmer/guile-wm#4]
How do you develop Guile WM? Do you use Xnest or some X server facility to have a window with the development version of gwm or do you use geiser? or both? Please enlighten me :)

**Comment by mwitmer** [comment]
Thanks for asking! I just did it in Geiser. I started by translating the
Xlib code for TinyWM into Scheme to get a really basic WM to work in and
then I added more features from there editing code and loading it into
the process with the occasional restart. It was a lot like writing emacs
extensions.

I wish I had more time to work on it at the moment but I've been short on
time lately. Hopefully I'll be able to come up with a few new little
features soon.

Thanks
Mark Witmer
